packages/@o3r/dynamic-content/README.md

```diff
@@ -38,19 +38,19 @@ A pipe to be used in your component templates:
 
 ```html
 <img src="{{'assets-otter/imgs/logo.png' | dynamicContent}}" /> or
-<img [src]="'assets-otter/imgs/logo.png' | dynamicContent" />
+<img ngSrc="{{'assets-otter/imgs/logo.png' | dynamicContent}}" fill/>
 ```
 
 and a service to be used in your component classes, for example:
 
 ```typescript
-
+import { firstValueFrom } from 'rxjs';
 @Component({
   /* ... */
 })
 export class MyComponent {
   constructor(private service: DynamicContentService) {
-    const imgSrc = this.service.getMediaPath('assets/assets-otter/imgs/logo.png');
+    const imgSrc = await firstValueFrom(this.service.getMediaPathStream('assets/assets-otter/imgs/logo.png'));
```

Comment:
constructor could not be asynchronous

Reply:
why not using the `async` pipe in the template ?

Reply:
It was more to store the imgSrc in the typescript (to have the same example than in the previous version of the doc) but I can definitely move it in the onInit

Reply:
I a bit agree with Matthieu, this it not the purpose of the example, but we should not encourage to convert it to promise which would block the rules engine assets override action.
I am a bit scare that people will take this example to use it in their code and get issue because of it :S

Reply:
I believe the `ngOnInit` shouldn't be asynchronous either
also the `await` won't compile if not inside an `async` function

Reply:
We already have an example for the async pipe, this is meant to get the value in the typescript, maybe I can put it in a dedicated async function:
`async getImgSrc() {
  return await firstValueFrom(this.service.getMediaPathStream('assets/assets-otter/imgs/logo.png'));
}`
Would it be a good tradeoff ?

Reply:
Unfortunately, for me it is still not supporting the Rules Engine and should not be an example that we should provide :S

Reply:
I'll update to put the same one than the proposal you did on my other PR :)